.github/workflows/build.yaml

```diff
@@ -19,13 +19,13 @@ jobs:
             runner: macos-latest-xlarge
             system: ARM64-macOS
           - nix-system: x86_64-darwin
-            runner: macos-13-large
+            runner: macos-latest-xlarge
             system: X64-macOS
           - nix-system: aarch64-linux
             runner: namespace-profile-default-arm64
             system: ARM64-Linux
           - nix-system: x86_64-linux
-            runner: ubuntu-22.04
+            runner: UbuntuLatest32Cores128G
```

Comment:
Does this repo really benefit from the larger (more expensive) runners?

Reply:
On what basis do we pay for these? By minute or some such? I'm not aware of what our plan provides.